mono/mini/debugger-agent.c

```diff
@@ -7553,12 +7610,32 @@ event_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 		if (req->event_kind == EVENT_KIND_BREAKPOINT) {
 			g_assert (method);
 
-			req->info = mono_de_set_breakpoint (method, location, req, error);
-			if (!is_ok (error)) {
-				g_free (req);
-				PRINT_DEBUG_MSG (1, "[dbg] Failed to set breakpoint: %s\n", mono_error_get_message (error));
-				mono_error_cleanup (error);
-				return ERR_NO_SEQ_POINT_AT_IL_OFFSET;
+			burst_lock();
+			if (domain == g_BurstDebugDomain)
+			{
+				if (g_BurstDebugCallbacks.BurstBreakpointNotify) {
+
+					MonoBreakpoint* bp = g_new0 (MonoBreakpoint, 1);
```

Comment:
No, ownership remains with mono debugger, its released the same way as other breakpoints, I just only needed the allocation, rather than the rest of the contents of the mono_de_set_breakpoint, mono_de_clear_breakpoint will be called as usual to discard the allocation.